### PR TITLE
Refactor/clickable to fc

### PR DESCRIPTION
**Pull request by vishnusnathann**
# Description

Refactored  Clickabable component to FC.

## Which issue(s) does this PR relate to?

https://github.com/dcos-labs/ui-kit/issues/782

## Checklist

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] This PR is associated with a JIRA and mentions in the commit message footer ("Closes …")
- [ ] This PR contains breaking changes and states in the commit message body ("BREAKING CHANGE: …")
- [X] I have reviewed the changes and provided detail to the sections above

### CODE REVIEW

**Review by nataliepina — COMMENTED** [review]
Great job! I added one suggested change. I did a regression test and the story has not changed. 👍🏻

File: packages/clickable/components/clickable.tsx [hunk 1]
```diff
@@ -29,44 +29,30 @@ export interface ClickableProps {
...
-      onKeyPress: this.handleKeyPress,
-      ["data-cy"]: dataCy
-    });
-  }
+export const Clickable: React.FC<ClickableProps> = ({
+  tabIndex = -1,
+  role = "button",
+  disableFocusOutline = false,
+  ...props
+}) => {
```
[the remark this answers is not in the document]
Comment (vishnusnathann) [reply packages/clickable/components/clickable.tsx:37]: Thank you @nataliepina  will update my PR.

[the remark this answers is not in the document]
Comment (vishnusnathann) [reply packages/clickable/components/clickable.tsx]: @nataliepina Updated the code.

File: packages/clickable/components/clickable.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ export interface ClickableProps {
   /**
    * Children should be a HTML element.
    */
-  children: React.ReactElement<HTMLElement>;
+  children: React.ReactElement<HTMLElement> & React.ReactNode;
```
Comment (nataliepina) [inline packages/clickable/components/clickable.tsx:9]: Was there a type error that came up requiring you to add this `& React.ReactNode`?

Comment (vishnusnathann) [reply packages/clickable/components/clickable.tsx:9]: Yes

**Review by clintonmedbery — APPROVED** [review]
Thanks for your work! Seems to be working great!

**Pull request merged**